.github/workflows/CI.yml

```diff
@@ -39,6 +39,25 @@ jobs:
         run: npx envinfo
       - name: Build
         run: ./configure && make -j8
+  build-from-tarball:
```

Comment:
do we want to run this on windows / linux too?

Reply:
@MylesBorins is the tarball different per OS?  Looking at release jobs it looks like it is only built once.

Reply:
I added runs for windows and linux.

Reply:
I am -1 on windows.